05_ANALYSIS_TOOLS/etl_pipeline/transform/business_metrics.py

```
@@ -0,0 +1,637 @@
+                # Log comprehensive expense analysis
+                logger.info(f"Comprehensive expense analysis - Top 10 expense categories:")
+                sorted_expenses = sorted(all_expense_categories.items(), key=lambda x: x[1], reverse=True)
+                for expense_name, total_amount in sorted_expenses[:10]:
+                    logger.info(f"  {expense_name}: ${total_amount:,.2f}")
+                
+                return adjusted_monthly_ebitda
+            else:
+                logger.warning("No valid P&L data found for EBITDA calculation")
+                return None
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)